src/test/resources/org/checkstyle/autofix/recipe/upperell/before/InputUpperEllTest.java

```diff
@@ -0,0 +1,8 @@
+package org.checkstyle.autofix.recipe.upperell.before;
+
+public class InputUpperEllTest {
+    long decimal = 12345l;
+    long hex = 0xABCDl;
+    long octal = 511L;
+    long binary = 0b1010l;
```

Comment:
Please add few test cases with String literals, and char variables with values like 'l'